calculations/do_calculations.py

```diff
@@ -0,0 +1,57 @@
+def temp_given_month(data):
+    """looks for highest temperature/humidity in a given year and month
+
+    Args:
+        data ([type]): [description]
+    """
+    year = input("Enter Year for Avg temperature and humidity as YYYY ")
+    month = input("Enter Month for Avg temperature and humidity as MM ")
```

Comment:
We were supposed to take command-line arguments from the user not in this way. Refactor the code as per requirement.